### ISSUE TITLE
[Feature Request] Add an aggressive blocking mode similar to desktop version

### ISSUE THREAD

**Issue by tankmohit** [brave/brave-ios#2737]
## Problem Description 
Aggressive mode on desktop is really awesome. Been using it for a while and never ran into any site breakage issue. It allowed me to remove other adblocking extensions such as ublock origin. I would happy to see the same feature on iOS version as many websites has pop-up ads and video player ads which can be blocked in safari using Adguard content blocker.

## Feature Overview 
Add an option to set shields mode to aggressive similar to desktop

**Comment by hennrynusasatria** [comment]
I agree with feature request from @tankmohit as above, but it seems to take higher efforts for this implementation.

Thank you.

**Comment by underling3311** [comment]
Please add this, ads still go right through brave's ad blocker!
